lib/screens/my_therapies_screen/add_therapy_screen.dart

```diff
@@ -71,6 +71,10 @@ class _AddTherapyScreenState extends State<AddTherapyScreen> {
       return false;
     }
 
+    if (_name!.isEmpty || _name!.trim() == '') {
+      return false;
+    }
+
```

Comment:
```suggestion

    if (_therapy!.isEmpty || _therapy!.trim() == '') {
      return false;
    }
```
Maybe would be good to apply the same rule to therapy as well.